src/Home.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  Image,
+  SafeAreaView,
+  TouchableOpacity,
+} from 'react-native';
+import { MaterialIcons } from '@expo/vector-icons';
+import cake from '../assets/cake.png';
+
+const styles = StyleSheet.create({
+  background: {
+    paddingTop: 10,
+    justifyContent: 'center',
+  },
+  container: {
+    flex: 1,
+    backgroundColor: '#e8cbf2',
+  },
+  Heading: {
+    fontSize: 40,
+    textAlign: 'center',
+  },
+  Heading2: {
+    fontSize: 20,
+    textAlign: 'left',
+  },
+
+  Image1: {
+    height: 200,
+    alignSelf: 'center',
+    width: 150,
+  },
+});
+
+const Home = ({ navigation }) => {
+  return (
+    <SafeAreaView style={styles.container}>
+      <View style={{ marginLeft: 20, marginTop: 50 }}>
+        <TouchableOpacity onPress={() => navigation.openDrawer()}>
+          <MaterialIcons name="menu" size={24} color="black" />
+        </TouchableOpacity>
+      </View>
```

Comment:
I see menu in this page only. It should be available in other pages as well, isn't it?